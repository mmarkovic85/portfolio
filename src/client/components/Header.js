import React from "react";

const Header = () => (
  <header>
    <h6>Header placeholder</h6>
  </header>
);

export {
  Header as default
};
