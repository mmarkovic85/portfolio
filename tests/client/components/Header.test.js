import React from "react";
import renderer from "react-test-renderer";
import Header from "../../../src/client/components/Header";

const NavbarPlaceholder = () => (<div>navbar placeholder</div>);

test("Should render Header component correctly", () => {
  const rendered = renderer.create(
    <Header navbar={NavbarPlaceholder} />
  );

  expect(rendered.toJSON()).toMatchSnapshot();
});
