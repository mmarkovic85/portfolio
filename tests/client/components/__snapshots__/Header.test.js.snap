// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Header component correctly 1`] = `
<header>
  <h6>
    Header placeholder
  </h6>
  <div>
    navbar placeholder
  </div>
</header>
`;
